src/modules/0.0.7/aws_route53_hosted_zones/entity/resource_records_set.ts

```diff
@@ -27,13 +28,13 @@ export enum RecordType {
   TXT = "TXT",
 }
 
-// TODO add unique constraint based on name + record type + parenthostedzone
+@Unique('UQ_name__record_type', ['name', 'recordType'])
```

Comment:
This unique declaration doesn't include the parenthostedzone but the comment called it out. Why is it not actually necessary?